### PR TITLE
151

### PR DESCRIPTION
**Pull request by croqueGrec09**

### CODE REVIEW

File: app/grails-app/views/templates/subscription/_copyElements.gsp [hunk 1]
```diff
@@ -305,42 +306,55 @@
...
-                <td  name="subscription.takeOwner.source">
+                <td name="subscription.takeLicenses.source">
                     <div>
                         <b><i class="balance scale icon"></i>${message(code: 'license.label')}:</b>
-                        <g:if test="${sourceSubscription.owner}">
-                            <g:link controller="license" action="show" target="_blank" id="${sourceSubscription.owner.id}">
-                                ${sourceSubscription.owner}
+                        <g:each in="${sourceLicenses}" var="license">
+                            <g:link controller="license" action="show" target="_blank" id="${license.id}">
+                                <div value="${GenericOIDService.getOID(license)}" class="la-multi-sources">
```
Comment (klodav) [inline app/grails-app/views/templates/subscription/_copyElements.gsp:314]: **value** is not an allowed attribute here; please use custom **data attributes**

File: app/grails-app/views/templates/subscription/_copyElements.gsp [hunk 2]
```diff
@@ -305,42 +306,55 @@
...
+                <td  name="subscription.takeLicenses.target">
                     <div>
                         <b><i class="balance scale icon"></i>${message(code: 'license.label')}:</b>
-                        <g:if test="${targetSubscription?.owner}">
-                            <g:link controller="license" action="show" target="_blank" id="${targetSubscription?.owner?.id}">
-                                ${targetSubscription?.owner}
-                            </g:link>
-                        </g:if>
+                        <g:each in="${targetLicenses}" var="license">
+                            <div value="${GenericOIDService.getOID(license)}">
```
Comment (klodav) [inline app/grails-app/views/templates/subscription/_copyElements.gsp:341]: **value** is not an allowed attribute here; please use custom **data attributes**

File: app/grails-app/views/templates/subscription/_copyElements.gsp [hunk 3]
```diff
@@ -351,7 +365,7 @@
                         </g:if>
                         <g:each in="${source_visibleOrgRelations}" var="source_role">
                             <g:if test="${source_role.org}">
-                                <div value="${genericOIDService.getOID(source_role)}" class="la-multi-sources">
+                                <div value="${GenericOIDService.getOID(source_role)}" class="la-multi-sources">
```
Comment (klodav) [inline app/grails-app/views/templates/subscription/_copyElements.gsp:368]: **value** is not an allowed attribute here; please use custom **data attributes**

File: app/grails-app/views/templates/subscription/_copyElements.gsp (hunk 1 above)
Comment (croqueGrec09) [reply app/grails-app/views/templates/subscription/_copyElements.gsp]: Fixed in 5de1adb

File: app/grails-app/views/templates/subscription/_copyElements.gsp (hunk 3 above)
Comment (croqueGrec09) [reply app/grails-app/views/templates/subscription/_copyElements.gsp]: Fixed in 5de1adb

File: app/grails-app/views/templates/subscription/_copyElements.gsp (hunk 2 above)
Comment (croqueGrec09) [reply app/grails-app/views/templates/subscription/_copyElements.gsp]: Fixed in 5de1adb

**Pull request merged**